substrate/frame/contracts/fixtures/Cargo.toml

```diff
@@ -14,11 +14,12 @@ sp-runtime = { path = "../../../primitives/runtime" }
 anyhow = "1.0.0"
 
 [build-dependencies]
+polkavm-derive = "0.2.0"
```

Comment:
We don't need this in our build.rs file? Only the fixtures itself depend on it.

Reply:
The idea is that this package gets pulled into the cache? I would rather just remove `--offline` for the fixture build. I don't see why this is necessary as all the other builds will still need network access.

Reply:
I would prefer that if that is okay. It is really confusing to me to put things in a manifest just for its side effect.